src/ProjectMSDLApplication.cpp

```diff
@@ -160,6 +160,10 @@ void ProjectMSDLApplication::defineOptions(Poco::Util::OptionSet& options)
                              false, "<number>", true)
                           .binding("projectM.beatSensitivity", _commandLineOverrides));
 
+    options.addOption(Option("textureSharingEnabled", "", "Syphon texture sharing enabled.",
```

Comment:
Same here - only provide the command-line option in builds which actually have Syphon support built-in. for any other build, it would confuse users.